assets/css/edd-admin.css

```diff
@@ -596,63 +596,21 @@ textarea[name="edd-note"] {
 
 .edd-amount-type-wrapper {
 	position: relative;
-	display: inline-block;
-}
-
-.edd-amount-type-select-wrapper {
-	position: absolute;
-	width: 45px;
-	background: #fafafa;
-	opacity: 0.9;
-	top: 1px;
-	left: 1px;
-	bottom: 1px;
-	border-top-left-radius: 4px;
-	border-bottom-left-radius: 4px;
-}
-
-.edd-amount-type-wrapper .edd-amount-type-select-wrapper:after {
-	display: block;
-	width: 17px;
-	height: 17px;
-	position: absolute;
-	right: 2px;
-	top: 3px;
-	font-family: dashicons;
-	font-size: 15px;
-	content: '\f140';
-	color: #555;
-	margin-top: 1px;
-	z-index: 2;
+	display: flex;
 }
 
 .edd-amount-type-wrapper select {
-	line-height: 1;
-	text-align: center;
-	border: 0;
-	background: transparent;
-	width: 45px;
-	min-height: 0;
-	height: 100%;
-	margin: 0;
-	padding-left: 8px;
-	border-right: 1px solid #ddd;
-	-webkit-appearance: none;
-	border-top-right-radius: 0;
-	border-bottom-right-radius: 0;
-	box-shadow: none;
-	position: relative;
-	z-index: 3;
-}
-
-.edd-amount-type-wrapper input {
-	margin: 0;
+	background-color: #fafafa;
```

Comment:
I think we should remove this too so we aren't overriding any core input styles besides the border radius.